neurolib/models/multimodel/model.py

```diff
@@ -205,17 +196,14 @@ def integrate(self, append_outputs=False, simulate_bold=False, noise_input=None)
             return_xarray=True,
         )
         self.storeOutputsAndStates(result, append=append_outputs)
-        # force bold if params['bold'] == True
-        if self.params.get("bold", False):
-            simulate_bold = True
 
         # bold simulation after integration
         if simulate_bold and self.boldInitialized:
             self.simulateBold(result[self.default_output].values.T, append=True)
 
     def setInitialValuesToLastState(self):
-        # set start t for next run for the last value now
-        self.start_t = self.t[-1]
+        if not hasattr(self, "t"):
+            raise ValueError("You tried using continue_run=True on the first run.")
```

Comment:
```suggestion
            raise ValueError("You tried using `continue_run=True` on the first run.")
```

Reply:
thank you Vasco, I forgot to make the suggested change in the multimodel